components/Room.tsx

```diff
@@ -142,12 +145,23 @@ function Room({
             />
           </Box>
         )}
+
+        {state.status === 'connected' && isInviteParticipantVisible && (
+          <Box width='medium' fill='vertical'>
+            <InviteParticipant
+              roomId={roomId}
+              onChangeInviteParticipantVisible={setIsInviteParticipantVisible}
+            />
+          </Box>
+        )}
       </Box>
       {state.status === 'connected' && (
         <>
           <RoomControls
             isParticipantsListVisible={isParticipantsListVisible}
+            isInviteParticipantVisible={isInviteParticipantVisible}
             onChangeParticipantsListVisible={setIsParticipantsListVisible}
+            onChangeInviteParticipantVisible={setIsInviteParticipantVisible}
```

Comment:
I'm not sure about naming this `onChangeInviteParticipantVisible` as this is not an event. We can pass this in as a prop that's named `setIsInviteParticipantVisible` since that's what it's doing. WDYT?

Reply:
I agree. I'm changing the prop name above this as well, since both are used to toggle components visibility.

Resolved in [869081c](https://github.com/team-telnyx/telnyx-meet/pull/70/commits/869081c770edf787c559b3375418348e82b4d0c2)